packages/config/src/index.ts

```diff
@@ -94,13 +94,15 @@ export class ConfigLoader<Schema extends z.ZodTypeAny> {
 
   async loadAndValidate(sources: ConfigSource[] = []) {
     let config = this.schema.parse({});
+    // If the config setting is an array, override instead of merge
+    const mergeRule = (_obj: any, src: any) => (_.isArray(src) ? src : undefined);
```

Comment:
I prefer to use native JS where possible.

```suggestion
    const mergeRule = (_obj: any, src: any) => (Array.isArray(src) ? src : undefined);
```